src/components/DialogHeader/DialogHeader.tsx

```diff
@@ -1,28 +1,12 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
```

Comment:
DialogueHeader close button should either be a Fondue component or have correct focus styling